### PR TITLE
replace all solutions with a single .sln file

### PR DESCRIPTION
**Pull request by martinhenningjensen**

### CODE REVIEW

File: domains/auth/Auth.API/Integration.Tests/Integration.Tests.csproj [hunk 1]
```diff
@@ -43,4 +43,8 @@
         </None>
     </ItemGroup>
 
+    <ItemGroup>
+      <Compile Remove="IntegrationTestCollection.cs" />
```
Comment (TopSwagCode) [inline domains/auth/Auth.API/Integration.Tests/Integration.Tests.csproj:47]: Why?

File: domains/transfer/Shared/ProjectOriginClients.Tests/ProjectOriginClients.Tests.csproj [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.10.0" />
+    <PackageReference Include="xunit" Version="2.8.1" />
```
Comment (TopSwagCode) [inline domains/transfer/Shared/ProjectOriginClients.Tests/ProjectOriginClients.Tests.csproj:11]: Are we sure we want to run these?

File: domains/auth/Auth.API/Integration.Tests/Integration.Tests.csproj (hunk 1 above)
Comment (martinhenningjensen) [reply domains/auth/Auth.API/Integration.Tests/Integration.Tests.csproj:47]: Must be a lefover from my attempt to optimize Auth integration tests. I thought i got it all reverted.

File: domains/transfer/Shared/ProjectOriginClients.Tests/ProjectOriginClients.Tests.csproj (hunk 2 above)
Comment (martinhenningjensen) [reply domains/transfer/Shared/ProjectOriginClients.Tests/ProjectOriginClients.Tests.csproj:11]: I don't know... I found an unused test project, so I guess it's either run these or delete the project.

**Review by TopSwagCode — DISMISSED** [review]

**Review by ckr123 — DISMISSED** [review]

**Review by TopSwagCode — APPROVED** [review]

**Pull request merged**